ui/v2.5/src/index.scss

```diff
@@ -56,40 +80,84 @@ code {
 
 @media (min-width: 576px) {
   .zoom-0 {
-    width: 15rem;
+    width: 240px;
+
+    & .scene-card-video {
```

Comment:
The ampersand is implied when two styles are nested, so it's not strictly speaking necessary.

Reply:
Addressed this and optimised a little after doing some reading up on sass.